### ISSUE TITLE
MD051 - Allow for template strings

### ISSUE THREAD

**Issue by 0xdf223** [rvben/rumdl#30]
I would love to use MD051 to validate local links, but I have a ton of references to other posts that look like this:

```md
---
layout: post
title: "TITLE"
---

# Title

## Heading

This is a [test link]({% post_url another-post %}#anchor)
```

This triggers MD051:

```
$ rumdl check test4.md -e MD051
test4.md:10:11: [MD051] Link anchor '#anchor' does not exist in document headings

Issues: Found 1 issues in 1 file (6ms)
```

One solution would be to only validate anchors at the start of the reference. Or make that an configuration option.

**Comment by 0xdf223** [comment]
This seems to be resolved as of 0.0.111.

**Comment by rvben** [comment]
Thanks for confirming, @0xdf223 !
